package.json

```diff
@@ -33,7 +33,8 @@
     "update-protocol-d-ts": "node utils/protocol-types-generator update",
     "compare-protocol-d-ts": "node utils/protocol-types-generator compare",
     "test-install": "scripts/test-install.sh",
-    "generate-docs": "npm run tsc && api-extractor run --local --verbose && api-documenter markdown -i temp -o new-docs"
+    "generate-docs": "npm run tsc && api-extractor run --local --verbose && api-documenter markdown -i temp -o new-docs",
+    "ensure-new-docs-up-to-date": "npm run generate-docs && exit `git status --porcelain | head -255 | wc -l`"
```

Comment:
This would be nicer, but doesn't error if there's an untracked file in the repo. e.g.:

```
// start with clean repo
touch foo.txt
git diff-index --quiet HEAD exits with 0 
```